hyperjumptech/monika
# Chore: Add cancel workflow and format .github files

# Monika Pull Request (PR)
This PR resolves !1169

## What feature/issue does this PR add
1. Cancels workflow `.github/workflows/node.js.yml` if one of the job fails

## How did you implement / how did you fix it
1. Added workflow cancel action `andymckay/cancel-action@0.3` to every job
2. Removed `.github` from `.prettierignore`
3. Format files in `.github` directory

## How to test
1. Make one of run commands fail
2. See workflow get canceled when one of the jobs fail

## Screenshot
![image](https://github.com/hyperjumptech/monika/assets/9563873/4f4cd088-3423-4216-8ff7-269ac0529e12)
